### PR TITLE
add documentation for the new installation

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

File: setup_robotino/SetupKenny.asciidoc [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+/home /u none bind 0 0
+----
+
+Now we can mount the drive
+
+----
+sudo mount /u
+----
+
+Activate +STATD+ in +/etc/default/nfs-common+ by changing th +NEED_STATD+ to yes
```
Comment (bajo) [inline setup_robotino/SetupKenny.asciidoc:67]: typo th -> the

File: setup_robotino/SetupKenny.asciidoc [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+Enable passwordless login to all robot pcs for robot user:
+
+----
+ssh-keygen
+ssh-copy-id robotino
+ssh-copy-id scrat
+----
+
+For root user it is necessary allow login as root over ssh session, open the file +/etc/ssh/sshd_config+ and set +PermitRootLogin yes+ ,then enable passwordless login:
```
Comment (bajo) [inline setup_robotino/SetupKenny.asciidoc:143]: Should we do the following to disable root login with passwords altogether?
`PasswordAuthentication yes
PermitRootLogin without-password`

**Review by bajo — APPROVED** [review]
Just two minor comments. The rest looks good to me.

File: setup_robotino/SetupKenny.asciidoc (hunk 2 above)
Comment (NadiaHG) [reply setup_robotino/SetupKenny.asciidoc:143]: I have never tried with the parameter `PasswordAuthentication yes` , I will test it.

Thanks for the recommendation

File: setup_robotino/SetupKenny.asciidoc [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+filegen loopstats file loopstats type day enable
+filegen peerstats file peerstats type day enable
+filegen clockstats file clockstats type day enable
+
+# Specify one or more NTP servers.
+
+# Use servers from the NTP Pool Project. Approved by Ubuntu Technical Board
+# on 2011-02-08 (LP: #104525). See http://www.pool.ntp.org/join.html for
+# more information.
+server 10.0.1.2
```
Comment (NadiaHG) [inline setup_robotino/SetupKenny.asciidoc:219]: redefine the ip addresses

File: setup_robotino/SetupKenny.asciidoc [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+# Specify one or more NTP servers.
+
+# Use servers from the NTP Pool Project. Approved by Ubuntu Technical Board
+# on 2011-02-08 (LP: #104525). See http://www.pool.ntp.org/join.html for
+# more information.
+server 0.de.pool.ntp.org
+
+server  127.127.1.0     # local clock
+fudge   127.127.1.0 stratum 10
+restrict 10.0.1.2 mask 255.255.255.0 nomodify notrap
```
Comment (NadiaHG) [inline setup_robotino/SetupKenny.asciidoc:185]: redefine the ip addresses

File: setup_robotino/SetupKenny.asciidoc [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+
+The +/etc/network/interfaces+  should looks:
+
+----
+auto lo
+iface lo inet loopback
+
+auto eth0
+iface eth0 inet static
+address 10.0.1.1 (?)
```
Comment (NadiaHG) [inline setup_robotino/SetupKenny.asciidoc:38]: redefine the ip addresses

**Pull request merged**